src/context/GlobalContext.js

```diff
@@ -0,0 +1,10 @@
+import { createContext } from "react";
+
+export default createContext({
+  isModeAdmin: false,
+  isPanelReduce: false,
```

Comment:
isPanelHidden
```suggestion
  isPanelReduce: false,
```

ou encore, tu as aussi : "isPannelCollapsed"